doc: add documentation for firehose local setup (#181)
odpf/firehose: docs/docs/contribute/development.md

- A time-series database \(e.g. InfluxDB\) to store the metrics
- GUI visualization dashboard \(e.g. Grafana\) for detailed visualisation of metrics

> # Running locally

Reviewer: @eyeofvinay This should be second-level heading. `##`

> - The following guide/walk through provides a simple way to run firehose locally, for log sink as an example.

Reviewer: ```suggestion
The following steps provides a simple way to run firehose with log sink locally.
```

> - The guide uses the TestMessage (src/test/proto/TestMessage.proto) proto schema, which has already been provided for testing purposes.

Reviewer: ```suggestion
It uses the TestMessage (src/test/proto/TestMessage.proto) proto schema, which has already been provided for testing purposes.
```
